### DISCUSSION TITLE
Pixel perfect brush

### DISCUSSION THREAD

**Discussion by Wojtowiczaz** [keiwando/toonsquid#4] [Ideas]
Hi, pixel perfect mode for brush could be a great feature for fellow pixel artists. Other pixel art apps which have it: aseprite, Pixaki, pixel studio

Great app dev, keep the good work!

**Comment by keiwando** [comment]
I will definitely look into this!

And thank you for the kind words!

**Comment by keiwando** [comment]
The 1.3 update is available now and comes with a "pixel perfect" brush setting and a new default pixel brush with that setting enabled.

Thank you for the suggestion!

**Comment by Wojtowiczaz** [comment]
Thanks for listening! I'm gonna check it out today. :)

Pon., 29 sie 2022, 18:51 użytkownik Keiwan Donyagard <
***@***.***> napisał:
